peps/converters.py

```diff
@@ -140,6 +150,21 @@ def convert_pep_page(pep_number, content):
     # Fix PEP links
     pep_content = BeautifulSoup(data['content'], 'lxml')
     body_links = pep_content.find_all("a")
+    # Fix highlighting code
```

Comment:
This chunk (from here to the next comment) is dealing with blocks with an explicit language, such as `.. code-block:: python` or `.. code-block:: c`.

More detail: the generated PEPs from https://github.com/python/peps has HTML like for these:

```html
<pre class="code python literal-block">
...
<pre class="code c literal-block">
```

And with no language specified (`::`) have:

```html
<pre class="literal-block">
```

This chunk finds all `<pre class="code ...">`, deletes those classes from the `pre`, and wraps them in a `<div class="highlight">` for later processing.